### ISSUE TITLE
New release including fix for qa script in Python 3?

### ISSUE THREAD

**Issue by jrderuiter** [simon-anders/htseq#24]
I am getting the following error in the qa script in Python 3 using version 0.7.2:

```{bash}
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/Julian/Anaconda/envs/imfusion-dev/lib/python3.5/site-packages/HTSeq/scripts/qa.py", line 148
    print i, "reads processed"
          ^
SyntaxError: Missing parentheses in call to 'print'
```

I saw that this issue was fixed in a recent commit. Will there be a new version that includes this fix?

**Comment by yhoogstrate** [comment]
Is it possible there is an old installation of the HTSeq library present? I don't think this error can be produced with the current version of HTSeq (https://github.com/simon-anders/htseq/blob/master/python3/HTSeq/scripts/qa.py#L145) - `print()` seems to be applied appropriately over the entire file

**Comment by iosonofabio** [comment]
Thanks Julian, that's a known regression that I fixed but not yet made a release for. Will try this week...

**Comment by jrderuiter** [comment]
@iosonofabio That would be great! Are you also planning to supply wheels for HTSeq? That would make HTSeq easier and faster to install for many people.

**Comment by iosonofabio** [comment]
I tried and failed for now. Wheels don't really support Linux which is my OS, and pysam does not offer Windows wheels either, so until pysam starts supporting it we probably won't either.

But if you want to try implementing it yourself, please keep me posted! Ideally we would have an AppVeyor CI that runs​and makes wheels as well :-)

**Comment by iosonofabio** [comment]
Sorry I forgot: another way to make installation faster and easier for people is non-wheel binaries, e.g. from your Linux distribution, Homebrew/macports, Conda, or whatever. But it's hard for me to cover all those cases...

**Comment by jrderuiter** [comment]
I have already added the most recent release in bioconda: https://github.com/bioconda/bioconda-recipes/pull/4571

**Comment by iosonofabio** [comment]
Awesome thanks! Does that cover both Python 2 and 3?

**Comment by jrderuiter** [comment]
Jep! (Except for the qa script in Python 3, which is why I was asking about a new release!) Currently bioconda builds for Python 2.7, 3.4 and 3.5 I think.

**Comment by iosonofabio** [comment]
Great thanks, does bioconda provide binaries for OSX?

**Comment by yhoogstrate** [comment]
Yes, they do

**Comment by iosonofabio** [comment]
qa script is fixed in 0.8.0, wheels is under way and another issue altogether
